packages/schema/README.md

```diff
@@ -0,0 +1,30 @@
+<div align="center">
+
+![Josh Logo](https://evie.codes/josh-light.png)
+
+# @joshdb/schema
+
+**An auto ensure middleware for Josh.**
+
+[![GitHub](https://img.shields.io/github/license/josh-development/middlewares)](https://github.com/josh-development/middlewares/blob/main/LICENSE)
+[![npm](https://img.shields.io/npm/v/@joshdb/schema?color=crimson&logo=npm&style=flat-square&label=@joshdb/schema)](https://www.npmjs.com/package/@joshdb/schema)
+
+</div>
+
+## Description
+
+A schema validation middleware for Josh.
+
+## Features
+
+- Written in TypeScript
+- Offers CommonJS, ESM and UMD bundles
+- Fully tested
+
+## Installation
```

Comment:
We probably need a link to docs in all readmes

Reply:
todo

Reply:
not in this PR